lib/cli-engine.js

```diff
@@ -256,6 +257,31 @@ function isErrorMessage(message) {
     return message.severity === 2;
 }
 
+
+/**
+ * Checks if a provided path is a directory and returns a glob string matching
+ * all files under that directory if so, the path itself otherwise.
+ *
+ * Reason for this is that `glob` needs `/**` to collect all the files under a
+ * directory where as our previous implementation without `glob` simply walked
+ * a directory that is passed. So this is to maintain backwards compatibility.
+ *
+ * Also makes sure all path separators are POSIX style for `glob` compatibility.
+ *
+ * @param {string} pathname The directory path to be modified
+ * @returns {string} The glob path or the file path itself
+ */
+function processPath(pathname) {
+    var newPath = pathname;
+
+    if (shell.test("-d", pathname)) {
```

Comment:
We simply don't append `/**` and use it as is, which matches the file. Isn't that the expected behavior?